heir_py/__init__.py

```diff
@@ -0,0 +1,3 @@
+from decorator import compile
```

Comment:
```suggestion
from .decorator import compile
```

README instructions work for me if making the relative path explicit here